grakn-core/src/main/java/ai/grakn/util/REST.java

```diff
@@ -198,6 +198,13 @@
             public static final String RELATIONS_JSON_FIELD = "relations";
             public static final String RESOURCES_JSON_FIELD = "resources";
         }
+
+        /**
+         * Json fields used to describe tasks
+         */
+        public static class Task {
```

Comment:
I have a sneaking suspicion that either this is missing keys or this should not exist at all. Surely we have many other keys on task related json representations?